README.md

```diff
@@ -17,3 +17,6 @@ This package defines:
       anchor up (`HE`) or down (`HB`) to the nearest hour
 * `Inclusivity`, which represents whether an `AbstractInterval` is open, half-open, or
   closed
+
+
+![plot](test/references/HE.png)
```

Comment:
I am not sure if this should be here.
I figured:
 1. I already have this image in the repo
 2. Having an image in the README might be nice.

There are proper docs on the plots in the actual documentation saying how to generate this.

Reply:
I think some context needs to be given

Reply:
I think we don't need it in the readme then.
It has context when shown in the docs
```suggestion
```